disperser/cmd/dataapi/flags/flags.go

```diff
@@ -71,6 +71,13 @@ var (
 		EnvVar:   common.PrefixEnvVar(envVarPrefix, "SUBGRAPH_OPERATOR_STATE_API_SOCKET_ADDR"),
 		Required: true,
 	}
+	SubgraphApiPaymentsAddrFlag = cli.StringFlag{
+		Name: common.PrefixFlag(FlagPrefix, "sub-payments-socket-addr"),
+		//We need the socket address of the subgraph payments api to pull the subgraph data from.
+		Usage:    "the socket address of the subgraph payments api",
```

Comment:
is this really a socket address? aka ip:port? Or is it a full URL with http endpoint mapping to specific satsuma resource?

Reply:
It's just a full URL of an http endpoint

Reply:
no need to do in this PR but austin has raised this in the past and I agree, "socket address" typically mans "ip:port" pair. At least in offchain code/comments we should mention its a full URL
<img width="524" height="95" alt="image" src="https://github.com/user-attachments/assets/fa37274c-0163-4882-9a1d-bdacf518f75f" />

Reply:
I think all we need to do here is change the usage text for these subgraph flags to
`Usage: "the URL of the subgraph payments api",` and we're golden.
<img width="498" height="106" alt="image" src="https://github.com/user-attachments/assets/fee923c4-5264-4da6-b6b4-5c805570db25" />

Reply:
fdedee733bf19d4fdd40792d75565a37c93d5b84